src/com/carolinarollergirls/scoreboard/defaults/DefaultStatsModel.java

```diff
@@ -0,0 +1,456 @@
+package com.carolinarollergirls.scoreboard.defaults;
+/**
+ * Copyright (C) 2008-2012 Mr Temper <MrTemper@CarolinaRollergirls.com>
+ *
+ * This file is part of the Carolina Rollergirls (CRG) ScoreBoard.
+ * The CRG ScoreBoard is licensed under either the GNU General Public
+ * License version 3 (or later), or the Apache License 2.0, at your option.
+ * See the file COPYING for details.
+ */
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.List;
+import java.util.Map;
+
+import com.carolinarollergirls.scoreboard.Clock;
+import com.carolinarollergirls.scoreboard.Position;
+import com.carolinarollergirls.scoreboard.ScoreBoard;
+import com.carolinarollergirls.scoreboard.Skater;
+import com.carolinarollergirls.scoreboard.Stats;
+import com.carolinarollergirls.scoreboard.Team;
+import com.carolinarollergirls.scoreboard.event.ConditionalScoreBoardListener;
+import com.carolinarollergirls.scoreboard.event.ScoreBoardEvent;
+import com.carolinarollergirls.scoreboard.event.ScoreBoardListener;
+import com.carolinarollergirls.scoreboard.model.StatsModel;
+
+public class DefaultStatsModel extends DefaultScoreBoardEventProvider implements StatsModel
+{
+  public DefaultStatsModel(ScoreBoard sb) {
```

Comment:
This file is indented with 2 spaces, where existing code is indented with 1 tab. The same applies to `StatsModel.java` and `DefaultStatsModel.Tests.java`.